Microsoft.Toolkit.Uwp.Services/Services/OneDrive/OneDriveItemExtension.cs

```diff
@@ -124,7 +124,7 @@ public static class OneDriveItemExtension
         /// <returns>a OneDrive item or null if the request fail</returns>
         public static async Task<bool> MoveAsync(this IOneDriveClient provider, HttpRequestMessage request, OneDriveStorageFolder destinationFolder, string desiredNewName, CancellationToken cancellationToken)
         {
-            OneDriveParentReference rootParentReference = new OneDriveParentReference();
```

Comment:
Should this class be marked obsolete and use the new class methods?

Reply:
weird - this was already done when i came to review.  so yes.